app/src/main/java/com/rob729/quiethours/util/StartAlarm.kt

```diff
@@ -49,7 +49,14 @@ class StartAlarm(appContext: Context, workerParams: WorkerParameters) :
             .setContentIntent(pi)
             .build()
         notificationManager.notify(1112, notification)
-
+        StoreSession.writeInt(AppConstants.BEGIN_STATUS, StoreSession.readInt(AppConstants.BEGIN_STATUS) + 1)
+        StoreSession.writeString(AppConstants.ACTIVE_PROFILE_NAME, profileName)
+        if (vibrate) {
+            StoreSession.writeInt(AppConstants.VIBRATE_STATE_ICON, 1)
+        } else {
+            StoreSession.writeInt(AppConstants.VIBRATE_STATE_ICON, 0)
+        }
+        StoreSession.writeString(AppConstants.END_TIME, profileEndTime!!)
```

Comment:
Is this null assertion required?

Reply:
@rob729 Yes, It was highlighting red without null assertion

Reply:
Okay.